docs/logging.md

```diff
@@ -0,0 +1,10 @@
+# Logging, including Errors
+
+In the "Spider Data Catalog" stage, any errors are written to a database table. 
+[See documentation on that stage for more](stage/spider-data-catalog.md).
```

Comment:
Link titles should relate to the subject at the other end of the link, for a11y reasons. Maybe "See the documentation for the [spider data catalogue stage]()"?